owickstrom/komposition
# Await streaming

**Description:**

This pull request should fix #48. @danbroooks, please try this branch if you have time.

The PR adds a 500ms delay before starting the GSteamer widget to let the FFmpeg HTTP server start the streaming. It's an ugly hack, but I haven't found any reliable way to parse the output from FFmpeg to make sure the server is started. Any future help and suggestions would be very welcome.

**Checklist:**

- [x] Doesn't duplicate any existing PR

**How to test:**

1. Create a new project
2. Import a video file
3. Add video clips from library
4. Preview the timeline, sequence, or parallel

The preview window should show on first try, streaming the entire video you've previewed.
